src/index.html

```diff
@@ -1,14 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8"></meta>
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"></meta>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"></meta>
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"></link>
+    <link rel="preconnect" href="https://fonts.gstatic.com"></link>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,100&display=swap"
+      rel="stylesheet"></link>
+    <link rel="stylesheet" href="./style.css"></link>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="https://www.moyo.ua/ua/">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      </a>
+      <nav class="nav__links">
+        <ul>
+          <li class="li__item">
+            <a href="#apple" class="is-active">APPLE</a>
+          </li>
+          <li class="li__item">SAMSUNG</li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list>
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">APPLE</a>
          </li>
          <li class="nav__item">SAMSUNG</li>
```
let's make more constistent classes which will be follow to BEM